src/db/migrations/31-zora-approvals.ts

```diff
@@ -28,12 +27,13 @@ const pmRecords = [
   '0x14214694847ba90da2e9b08b6de397bd06d5a626',
   '0x676777d784ebcc9d3ff3e28ce73757864a8333b4']
 
-
 export const up = async (knex: Knex) => {
-
-  const messages: CrdtMessage[] = pmRecords.map(address =>
-    getCrdtUpdateMessage<NftFactory>(Table.nftFactories, { id: address, approved: true, autoApprove: true })
-  )
+  const messages: any[] = pmRecords.map(address => ({
```

Comment:
can just use the defaultAdminAddress for these so that we can keep using getCrdtUpdateMessage instead of manually doing it?

Reply:
We've got the same recurring problem when retrofitting migrations here - the `signer` column doesn't yet exist so if we modify the `getCrdtUpdateMessage` signature, we break previous usages in our migrations. Alternatively we can shift the order of migrations to change migration history but that's also bad practice.

Ideal scenario for migrations is that they deal purely with data, so I think this is a positive adjustment towards decoupling our migrations from our general business logic and maintaining a more chronological migration path.

Reply:
oh right, sorry, sounds fine then